src/test/java/com/google/api/codegen/testdata/library_config.baseline

```diff
@@ -16,10 +16,12 @@ language_settings:
 interfaces:
 - name: google.example.library.v1.LibraryService
   collections:
+  - name_pattern: archives/{archif_path=**}/books/{book}
```

Comment:
Could take this dependency: https://github.com/atteo/evo-inflector
Otherwise, could build on existing implementation to cover this case